toolkit/tools/precacher/precacher.go

```diff
@@ -392,13 +395,13 @@ func precachePackage(pkg *repocloner.RepoPackage, packagesAvailableFromRepos map
 	}()
 
 	logger.Log.Debugf("Pre-caching '%s' from '%s'", fileName, url)
-	err = retry.Run(func() error {
+	_, err = retry.RunWithExpBackoff(func() error {
 		err := network.DownloadFile(url, fullFilePath, nil, nil)
 		if err != nil {
 			logger.Log.Warnf("Attempt to download (%s) failed. Error: %s", url, err)
 		}
 		return err
-	}, downloadRetryAttempts, downloadRetryDuration)
+	}, downloadRetryAttempts, downloadRetryDuration, failureBackoffBase, nil)
```

Comment:
instead of `nil` can you add a constant which makes its intent clear.
```suggestion
	}, downloadRetryAttempts, downloadRetryDuration, failureBackoffBase, nil)
```

Reply:
Sure, will do